src/main/java/com/appium/manager/ConfigFileManager.java

```diff
@@ -32,7 +32,8 @@ public static ConfigFileManager getInstance() {
                 Enumeration keys = prop.propertyNames();
                 while (keys.hasMoreElements()) {
                     String key = (String) keys.nextElement();
-                    configFileMap.put(key, prop.getProperty(key));
+                    String value = (null == System.getenv(key)) ? prop.getProperty(key) : System.getenv(key);
```

Comment:
here it set only a single value which is 'key' how can this override all the values?

Reply:
The CI build has 2 errors that I can see:
```
2018-12-19T12:13:52.9182900Z [WARNING] Error injecting: org.apache.maven.report.projectinfo.CiManagementReport
2018-12-19T12:13:52.9188710Z java.lang.NoClassDefFoundError: org/apache/maven/doxia/siterenderer/DocumentContent
...
...
2018-12-19T12:13:52.9865900Z [ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.3:site (default-site) on project AppiumTestDistribution: Execution default-site of goal org.apache.maven.plugins:maven-site-plugin:3.3:site failed: A required class was missing while executing org.apache.maven.plugins:maven-site-plugin:3.3:site: org/apache/maven/doxia/siterenderer/DocumentContent
2018-12-19T12:13:52.9967990Z [ERROR] -----------------------------------------------------

```

And
```
2018-12-19T12:13:58.2459340Z [ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.0.0:check (validate) on project AppiumTestDistribution: Failed during checkstyle execution: There is 1 error reported by Checkstyle 8.14 with /Users/vsts/agent/2.144.0/work/1/s/google-style.xml ruleset. -> [Help 1]
2
```

Don't think these errors are related to the PR, but please do help / guide on how I could dig deeper into this and resolve it